src/pages/ReviewForm/ReviewForm.tsx

```diff
@@ -23,16 +26,70 @@ const StyledTextField = styled(TextField)(({ theme }) => ({
 }))
 
 interface ReviewFormProps {
+  setReviews: (review: ReviewWithId[]) => void
   open: boolean
   toggleDrawer: (newOpen: boolean) => void
+  bookId: string
+  reviews: ReviewWithId[]
 }
 
 export const ReviewForm: React.FC<ReviewFormProps> = ({
+  reviews,
+  setReviews,
   open,
   toggleDrawer,
+  bookId,
 }) => {
   const matches = useMediaQuery('(min-width:700px)')
 
+  const [content, setContent] = useState('')
+  const [rating, setRating] = useState<number>(0)
+  const [seniorities, setSeniorities] = useState<string[]>([])
+  const [dateTime, setDateTime] = useState('')
+
+  const handleTextChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setContent(event.target.value)
+  }
+
+  useEffect(() => {
+    const today = new Date()
+    const month = today.getMonth() + 1
+    const year = today.getFullYear()
+    const formattedDate = `${year}-${
+      month < 10 ? '0' + month : month
+    }-01T00:00:00`
+    setDateTime(formattedDate)
+  }, [])
+
+  const seniorityToSnakeCase = (categories: string[]): string[] => {
+    return categories.map(category => category.toLocaleUpperCase())
+  }
+
+  const handleSubmit = () => {
+    const reviewData = {
+      rating,
+      seniorities: seniorityToSnakeCase(seniorities),
+      content,
+      dateTime,
+      bookId,
+    }
+
+    apiService
+      .addReview(bookId, reviewData)
+      .then(data => {
+        setReviews([...reviews, data])
+        console.log('Review added successfully:', data)
```

Comment:
Add [snackbars](https://mui.com/material-ui/react-snackbar/) instead of console logs both for success and for error